src/send/EnterAmount.tsx

```diff
@@ -0,0 +1,481 @@
+import { parseInputAmount } from '@celo/utils/lib/parsing'
```

Comment:
this file is a little hard to review - it is basically a copy + paste of SendEnterAmount, with the send specific logic removed and expected as props.